### ISSUE TITLE
[v2] Newly introduced index doesn't discover the fragments

### ISSUE THREAD

**Issue by koresar** [jimkyndemeyer/js-graphql-intellij-plugin#235]
## Version and Environment Details
__Operation system__: MacOS

__IDE name and version__: WebStorm 2019.1
Build #WS-191.6183.63, built on March 21, 2019
JRE: 1.8.0_202-release-1483-b39 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.4

__Plugin version__: 2.0

## Expected Behaviour

Fragments are recognised.

## Actual Behaviour

See red fragments

![image](https://user-images.githubusercontent.com/1622575/55701557-02d87a80-5a17-11e9-99fc-120af28b31bb.png)

## Steps to Reproduce / Link to Repo with Reproduction and Instructions

As recommended by @jimkyndemeyer I'm using the `language=GraphQL` WebStorm syntax highlight feature via a JS comment. See picture above.

Before releasing the stable `v2` it was working just fine with `v2-beta`.

There is the trend to avoid **any** client-side (browser) libraries to do GraphPL POST requests.
* [Tweet 1](https://mobile.twitter.com/HenrikJoreteg/status/1115338381678899200) - _"The assumption that you need a client side lib to use GraphQL is a myth that needs to die."_
* Library using `/* GraphQL */` to syntax highlight the GraphQL queries - https://github.com/jaydenseric/graphql-react#graphql-react-3
* Adding Apollo client-side modules to a web app is some additional 100KB
* All our GraphQL queries are securely encrypted by webpack at the build step. We ship Base64 encoded strings (not a plain GraphQL any more) inside our production bundle. Thus there is no sense using tools like `gql` on the client.

Our requests look like this:

```js
        const { login } = await store.dispatch(
            "graphql/mutation",
            // language=GraphQL
            {
                query: `mutation login($email:String! $pwd:String!) { login(email:$email pwd:$pwd) { success message } }`,
                variables: { email, pwd }
            }
        );
```

Where `dispatch` is Redux/Vuex [action](https://vuex.vuejs.org/guide/actions.html) dispatching.

**Comment by jimkyndemeyer** [comment]
Hi Vasyl.

Thanks for the detailed description and your help in testing the 2.0 release.

This is definitely a valid use case that should be supported, and unfortunately I missed it when I introduced the fragments index.

I'll get this fixed for the first bugfix release.

As a temporary workaround you can manually install the plugin from disk using the latest beta release at https://github.com/jimkyndemeyer/js-graphql-intellij-plugin/blob/v2/alpha-releases/JS%20GraphQL-2.0.0-beta-2.zip

Best regards,
Jim.

**Comment by jimkyndemeyer** [comment]
Fixed for the upcoming 2.0.1 release.
